apps/OboeTester/app/src/main/java/com/google/sample/oboe/manualtest/AudioStreamBase.java

```diff
@@ -81,6 +81,18 @@ String dump(int framesPerBurst) {
 
             return buffer.toString();
         }
+        /**
+         *Converts ints from JNI to human-readable stream state
```

Comment:
Add space between "* C".
Change "JNI" to "Oboe index". JNI is just the interface between Java and Native.